conformance-tests/Makefile

```diff
@@ -0,0 +1,49 @@
+GATEWAY_CLASS = nginx
+SUPPORTED_FEATURES = Gateway
+KIND_KUBE_CONFIG_FOLDER = $${HOME}/.kube/kind
+TAG = latest
+PREFIX = conformance-test-runner
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: Makefile ## Display this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: build-test-image
+build-test-image: ## Build conformance test image
+	docker build -t $(PREFIX):$(TAG) -f Dockerfile ..
+
+.PHONY: create-kind-cluster
+create-kind-cluster: ## Create a KinD cluster
+	kind create cluster --image kindest/node:v1.27.1
+	kind export kubeconfig --kubeconfig $(KIND_KUBE_CONFIG_FOLDER)/config
+
+.PHONY: install-nkg
+install-nkg: ## Install NKG with provisioner on configured KinD cluster
+	kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.7.0/standard-install.yaml && sleep 10
```

Comment:
Blind sleeps are never a great idea. I would prefer these steps live in a script that the Make target could call, and in that script we could poll until the gateway resources exist, rather than hoping they exist within our sleep period.

Reply:
If we're just waiting for a Pod to be running, we could use a simple `while` or `until` loop in a bash script and just call a `kubectl` command that returns the status of the resource we're waiting for.

Reply:
we're actually waiting for webhook container to start so I'll just do a while on the event for it